### ISSUE TITLE
Rhythmic movements of the tympanic membrane

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10146]
**Preferred term label:**
Rhythmic movements of the tympanic membrane 
and a second tab term which might deserve its own HPO code although it is a systematic documentation of the first term:
Sawtooth pattern on tympanometry

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tensor tympani syndrome
https://www.ncbi.nlm.nih.gov/books/NBK519055/
"while conducting an otoscopic exam, rhythmic movements of the tympanic membrane have been described"
"long-term tympanometry can be used. A saw-tooth-like pattern representing tympanic membrane movement can be seen, which supports the diagnosis of TT syndrome"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000043
- label: Rhythmic movements of the tympanic membrane
- parents: [Abnormal ear physiology (HP:0031704)](https://hpo.jax.org/app/browse/term/HP:0031704)
- pmid: PMID:30085597
- def: Regularly occuring motion of the tympanic membrane (eardrum), which is located between the inner and middle ear. This can be a manifestation of tensor tympani myoclonus, which can also be ascertained as sawtooth patterns on long-term tympanometry.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10146
- synonyms:
